.github/workflows/ci.yml

```
@@ -42,7 +37,7 @@ jobs:
         run: java --version
 
       - name: SonarCloud Scan
-        uses: SonarSource/sonarqube-scan-action@v4
+        uses: SonarSource/sonarqube-scan-action@v3
         env:
           SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)